### ISSUE TITLE
Semantics should have a flag for excludeSemantics

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#12433]
The Semantics widget should provide a flag to exclude all child semantics.

This would simplify the common use case were the Semantics widget is used to replace all child semantics.

Currently, this requires wrapping everything in a ExcludeSemantics widget.
